.github/workflows/godot_cpp_test.yml

```diff
@@ -24,6 +24,9 @@ jobs:
 
       - name: Setup Python and SCons
         uses: ./.github/actions/godot-deps
+        with:
+          # FIXME: godot-cpp build fails on 4.8.0 in this context specifically.
+          scons-version: 4.7.0
```

Comment:
This should be fixed now, see https://github.com/godotengine/godot-cpp/pull/1526.